components/core/src/os/filesystem/windows.rs

```diff
@@ -25,7 +26,11 @@ pub fn chmod(r_path: *const c_char, mode: u32) -> c_int {
         let path = CStr::from_ptr(r_path).to_str().unwrap();
         match Path::new(path).exists() {
             false => 1,
-            true => 0
-        } 
+            true => 0,
+        }
     }
 }
+
+pub fn symlink<P: AsRef<Path>, Q: AsRef<Path>>(src: P, dst: Q) -> io::Result<()> {
+    unimplemented!();
```

Comment:
Are Junctions still a thing in Windows? How do they achieve symlinks these days (if at all)?